test/n2z/environment.yaml

```diff
@@ -0,0 +1,12 @@
+name: papermill
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.8
+  - jupyter
+  - papermill
+  - pyproj
+  - numpy
+  - gdal=3.4.2
+  - netCDF4
```

Comment:
This was just copied, I bet I don't need them all

Reply:
Yeah - GDAL can definitely go (it's a big package, too, so it will help with build time to remove it). Right now, I think `pyproj`, `numpy` and `netCDF4` can also go.